src/pages/artikler/index.astro

```diff
@@ -0,0 +1,60 @@
+---
+import Layout from '@layouts/default.astro';
+import { getCollection } from 'astro:content';
+import PostCard from '@components/post-card.astro';
+import Brief from '@components/brief.astro';
+
+const posts = await getCollection('post');
+
+const compareNum = (a: any, b: any) => {
+  if (a.data.date.valueOf() > b.data.date.valueOf()) {
+    return -1;
+  } else if (a.data.date.valueOf() < b.data.date.valueOf()) {
+    return 1;
+  }
+  return 0;
+}
+---
+
+<Layout
+  title="Artikler"
+  description="Artikler vi har skrever"
+  socialMediaTitle="Artikler"
+  path="/artikler"
+>
+  <main>
+    <Brief 
+      title="Artikler"
+      content={[]}
+    />
+    <section class="posts">
+      {
+        posts
+          .sort(compareNum)
+          .filter(project => project.data.image)
+          .filter((p, index) => index < 12)
```

Comment:
Consider combining the filter operations into one in order to improve performance at the cost of slightly worse readability.
```suggestion
          .filter((project, index) => project.data.image && index < 12)
```